pwnlib/commandline/cyclic.py

```diff
@@ -20,7 +20,8 @@
 parser.add_argument(
     '-a', '--alphabet',
     metavar = 'alphabet',
-    default = string.ascii_lowercase,
+    default = string.ascii_lowercase.encode(),
+    type = bytes,
```

Comment:
Does this work on python 3, have you tested this?

Reply:
Yes. I tested it on both python2 and python3